ci-operator/step-registry/ipi/conf/gcp/private-dns-zone/ipi-conf-gcp-private-dns-zone-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${PRE_CREATE_CLUSTER_PVTZ}" != "yes" ]]; then
```

Comment:
I think `ipi-conf-gcp-private-dns-zone` does not need to judge `PRE_CREATE_CLUSTER_PVTZ`, when the step is called in workflow, then the author of the workflow should be responsible to call `gcp-provision-private-dns-zone` in previous steps to generate necessary resources for this step.